src/databricks/labs/ucx/installer/workflows.py

```diff
@@ -346,7 +346,9 @@ def for_cli(cls, ws: WorkspaceClient):
     def create_jobs(self, prompts):
         logger.debug(f"Creating jobs from tasks in {main.__name__}")
         remote_wheel = self._upload_wheel(prompts)
-        desired_steps = {t.workflow for t in self._tasks if t.cloud_compatible(self._ws.config)}
+        desired_steps = {
```

Comment:
rewrite it to a loop - it no longer fits the single line

Reply:
outdated